help_to_heat/templates/allauth/account/base.html

```diff
@@ -85,7 +85,7 @@
             <div>
               <ul class="govuk-list">
                 {% for message in messages %}
-                  <li>{{message}}</li>
+                  <li>{{message | safe}}</li>
```

Comment:
Are we definitely always in control of the message we're setting here?

Reply:
So to confirm, the message is only set from inside authentication_views.py and is set to the following values (line numbers included)
- Please enter an email and password. (59)
- Login failed - please check your credentials. If you believe they are correct... (69)
- Login failed - please accept the invitation you were sent via email. If you have... (65)
- Something has gone wrong.  Please contact your team leader. (74) (110) (165) (214) (226) (230)
- Please enter an email. (125)
- Passwords must match. (170) (298)
- **ValidationError output messages** (175) (310)
- Please fix the errors and try again (177)
- Password successfully set. (182)
- Please enter the otp (220)
- This link is not valid. It may have expired or have already been used. Please try again. (275) (280) (302)

the messages that derive from ValidationError are the only particularly scary ones, though from doing some inspection they do not provide user input directly. examples include:
- This password is too common.
- This password is entirely numeric.
- This password is too short. It must contain at least %(min_length)d character.

though if it's not worth the risk the email can be left as plain text

Reply:
I think I'd feel more comfortable if we left it as plain text. It's vanishingly unlikely, but this would leave us open to a supply chain attack where one of our dependencies (django or marshmallow by the looks of things) gets updated in such a way that we end up injecting some unsafe HTML. And of course the open source supply chain has been in the news this week with `xz`! Would need to be more carefully considered to do properly, and I don't think that's in scope for this ticket.